charts/posthog/templates/ingress.yaml

```diff
@@ -17,16 +17,6 @@ metadata:
     kubernetes.io/ingress.global-static-ip-name: {{ .Values.ingress.gcp.ip_name | quote }}
     {{- end }}
    {{- end }}
-   {{- if (eq (include "ingress.type" .) "alb") }}
```

Comment:
Why remove this as part of README changes?

Reply:
being lazy ...  because I was looking into this as I was writing up the minimum values files, but I can make it a separate PR.